<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>615</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="book_name">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_number">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>100</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Apple SD Gothic Neo</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Book Number</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="name_text">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="number_text">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>269</x>
      <y>150</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Book</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_result">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>311</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
